CHANGELOG.md

```diff
@@ -5,4 +5,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## v1.0.0-alpha - 09/27/2023
 
-Initial release of Arcadia-Science/noveltree, created with the [nf-core](https://nf-co.re/) template.
+Initial release of NovelTree.
+
+## v1.0.1-alpha - 09/28/2023
```

Comment:
Can you add this to the top. So the order is descending by date. So the most recent change is the easiest to find

Reply:
Ah, yup! Makes sense - wasn't sure what the best/standard convention was